### PR TITLE
Fixes #53 - Update references to point to RFC8942

### LINKED ISSUE #53: Update references to HTTP Client Hints to RFC8942

### PR DESCRIPTION
**Pull request by miketaylr**
PTAL @yoavweiss, thanks!

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -262,10 +252,10 @@ When asked to <dfn abstract-op>remove client hints from redirect if needed</dfn>
   <var>hintName</var> and if the result of running <a
   href="https://w3c.github.io/webappsec-permissions-policy/#algo-should-request-be-allowed-to-use-feature">Should
   request be allowed to use feature?</a>, given <var>request</var> and <var>hintName</var>’s
-  <a href="http://httpwg.org/http-extensions/client-hints.html#opt-in-via-feature-policy">associated
+  <a href="#policy-controlled-features">associated
```
Comment (miketaylr) [inline index.bs:255]: This change in particular (as well as the similar change just above) could use some extra eyeballs -- it seemed like a mistake, but I could be wrong!

Comment (yoavweiss) [reply index.bs:255]: I'm failing to find what this was pointing to... Changing the link as you did makes sense.

File: index.bs [hunk 2]
```diff
@@ -83,7 +74,7 @@ Infrastructure definition {#definition}
 The specification of the Client Hints **infrastructure** is divided between the
 following specifications and proposals:
 
-* IETF [[!CLIENT-HINTS]]
+* IETF [[!RFC8942]]
```
Comment (yoavweiss) [inline index.bs:77]: LGTM assuming bikeshed is happy

**Review by yoavweiss — APPROVED** [review]
LGTM

File: index.bs (hunk 2 above)
Comment (miketaylr) [reply index.bs:77]: Yeah, it was after I did `bikeshed update`. :)

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**